implementations/rust/ockam/ockam_core/src/lib.rs

```diff
@@ -39,6 +39,11 @@ pub extern crate hex;
 pub extern crate async_trait;
 pub use async_trait::async_trait as worker;
 
+pub extern crate ockam_macro;
```

Comment:
Why does this need to be `pub extern crate`? Ditto for `futures_util`.

Reply:
removed pub from both